docs/content/deploy.md

```diff
@@ -28,98 +24,58 @@ To start a local deployment, either:
-<br/> To do this:
-
-    1. Open the `spatialos.json` file in the root folder of the `Playground` project.
-    1. Change the `name` field so it matches the name of your SpatialOS project. You can find this in the SpatialOS [Console](https://console.improbable.io). It’ll be something like `beta_someword_anotherword_000`.
-1. Build and upload a game assembly for the deployment
-    - The assembly includes executable files for the client-workers and server-workers, and the assets both types of workers use (such as the models and textures used by the client-server - that is, the game executable code - to visualise the game).
-    1. To build an assembly; in the Unity Editor, select **SpatialOS** > **Build all workers for cloud**.
-
-    2. To upload an assembly; open a terminal and navigate to the `UnityGDK` directory (the repository you’ve cloned). Run `spatial cloud upload <assembly name>`.
-    - The `<assembly name>` is a label you create so you can identify this assembly in the next step - for example you could call it `MyGDKAssembly`.
-    
-    > **It’s finished uploading when:** You see `spatial upload <assembly name> succeeded` printed in your terminal output.
-1. Launch a cloud deployment
-    - In the same terminal window, run `spatial cloud launch <assembly name> cloud_launch.json <deployment name> --snapshot=snapshots/default.snapshot`
-    - This command defaults to deploying to clusters located in the US. So if you’re in Europe, add the `--cluster_region=eu` flag for lower latency.
-
-    > **It’s done when:** You see `Successfully created deployment` printed in your Unity Editor console output.
-
-1. Launch a game client:
-    1. Open the SpatialOS [Console](https://console.improbable.io/projects). You’ll see the project and the deployment you just created.
-    1. In the SpatialOS Console, select the deployment’s name to open the overview page.
-    1. Select **Launch**.
-        > You can ignore the prompt to install the Launcher, as it’s installed as part of the SpatialOS Installer.
-    1. To get links to share the game with others, select **Share**.
-    1. Once you’ve finished playing, select **Stop** in the Console.
+**About the `spatial cloud launch` command**   
+`spatial cloud launch` deploys a project to the cloud. Its full syntax is:
+
+ ```
+spatial cloud launch <assembly name> <launch configuration> <deployment name> --snapshot=<snapshot file>
+```
+
+where:
+
+  * `<assembly name>` is the name of the assembly the deployment will use (the one you named above).    
+  * `<launch configuration>` is the configuration file for the deployment. This project includes `default_launch.json`, which is intended for use with local deployments, and `cloud_launch.json`, which is for cloud deployments.
+  * `<deployment name>` is a name of your choice, which you’ll use to identify the deployment. Must be in lowercase.
```

Comment:
There are more rules about the name of the deployment than just lowercase. Can we link to another page in the docs for the full rules?

Reply:
Must be in lowercase.

->
The name must be all lowercase letters.

Reply:
oh yeah, fixed it somewhere else. but apparently not here